web/src/App.jsx

```
@@ -322,24 +477,98 @@ function App() {
+                <div
+                  key={`${result.node_id}-${idx}`}
+                  className="search-result-item"
+                  onClick={() => handleSearchResultClick(result)}
+                >
+                  <div className="search-result-meta">
+                    <span className="search-result-conv">{result.conversation_name}</span>
+                    <span className={`search-result-role ${result.role}`}>{result.role}</span>
+                  </div>
+                  <div
+                    className="search-result-snippet"
+                    dangerouslySetInnerHTML={{ __html: result.snippet }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)